File: Java/level 2/Find a decimal number.java
class NumOfPrime {
	int numberOfPrime(int n) {
		int answer = 0;
    int cnt=0;
	  for(int a=2;a<=n;a++){
      cnt=0;
  	  for(int b=1;b<=a;b++){
    		if(a%b==0)
          cnt++;
    }
    if(cnt==2)
      answer++;
  }
	return answer;
	}

	public static void main(String[] args) {
		NumOfPrime prime = new NumOfPrime();
		System.out.println( prime.numberOfPrime(10) );
	}

}

File: Python/level 2/Find a decimal number.py
def numberOfPrime(n):
    # 1부터 n사이의 소수는 몇 개인가요?
    answer = 0
    for a in range(2,n+1):
        cnt = 0
        for b in range(1,a+1):
            if a%b==0:
                cnt+=1
        if(cnt==2):
            answer+=1
    return answer


# 아래는 테스트로 출력해 보기 위한 코드입니다.
print(numberOfPrime(10))